src/Filter/FieldDefinitionAdapter/Checkbox.php

```diff
@@ -104,7 +104,7 @@ public function getIndexData($object)
     }
 
     /**
-     * @param array $fieldFilter
+     * @param string $fieldFilter
```

Comment:
```suggestion
     * @param bool $fieldFilter
```
As per the comment below